common/src/main/scala/explore/model/RootModel.scala

```diff
@@ -1,23 +1,92 @@
-// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
 package explore.model
 
+import cats.implicits._
 import cats.effect._
-
 import crystal._
+import clue.js.AjaxGraphQLClient
+import clue.js.WebSocketGraphQLClient
+import explore.graphql.TestQuery
+import io.lemonlabs.uri.Url
```

Comment:
It's just a `Uri`/`Url`library. I'd have liked to use `http4s`'s `Uri`, but AFAIK it's not available in Scala.js. I know you've done significant work in trying to have `http4s-client` on Scala.js. Do you think it would make sense (and it would be relatively easy) to extract `Uri` into a module?

Reply:
Oh wait, I don't know if you were referring to the `Url` or to `clue`. `clue` is a library where I moved the GraphQL client logic: https://github.com/rpiaggio/clue

Reply:
I was referring to `Url`, maybe http4s uri could be ported. Have you seen the one from `sttp`?

Reply:
I hadn't seen the one from `sttp`. They have a nice `model` library that implements this without the need of bringing in the whole client, I will explore it. Or, we could evaluate actually using their client.